crates/notary/server/tee/README.md

```diff
@@ -1,3 +1,114 @@
 This folder contains the necessary files to build a Docker image for running the Notary Server on Intel SGX-enabled hardware.
 
-The container is built as part of the CI pipeline. For details on the build process, refer to the [CI workflow configuration](../../../../.github/workflows/ci.yml).
+## Compile the Notary Server for Intel SGX
+
+We use [Gramine](https://github.com/gramineproject/gramine) to run the Notary Server on Intel SGX. Gramine allows the Notary Server to run in an isolated environment with minimal host requirements.
+
+The isolated environment is defined via the manifest template (`notary-server.manifest.template`).
+
+The Notary Server for SGX is compiled with the Rust feature flag `tee_quote`. This enables the server to use an ephemeral private notary key for signing attestations (`private_key_pem_path: "/ephemeral/notary.key"`) and also adds the SGX *quote* to the server's `/info` endpoint.
+
+### CI
+
+The [notary-server-sgx Docker container](https://github.com/tlsnotary/tlsn/pkgs/container/tlsn%2Fnotary-server-sgx) is built as part of the CI pipeline. For details on the build process, refer to the [CI workflow configuration](../../../../.github/workflows/ci.yml).
```

Comment:
nit: hmm isn't this already explained in the paragraph below? still feel its redundant imo...

```md
We also publish a Docker image `notary-server-sgx` at <https://github.com/tlsnotary/tlsn/pkgs/container/tlsn%2Fnotary-server-sgx>.
```